### PR TITLE
Added Controls / Controller Binds

### PR DESCRIPTION
**Pull request by avakimc**
- added controller binds for all actions except auto-align and auto-intake
- added driver_controller_port in RobotMap
- revised cancelScore usage
- remove outtake-related stuff (position_deg in Cal.java, OuttakeSequence.java)
- created rezeroGyro() method to drive
- changed some LED's to private
- added JoystickUtil.java

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/arm/Arm.java [hunk 1]
```diff
@@ -43,6 +43,7 @@ public enum ArmPosition {
   public ScoringLocationUtil scoreLoc;
   private ArmPosition desiredPosition = ArmPosition.STARTING;
   public boolean isScoring = false;
+  public boolean cancelledScore = false;
```
Comment (balghane) [inline src/main/java/frc/robot/subsystems/arm/Arm.java:46]: is `isScoring` used?

File: src/main/java/frc/robot/subsystems/arm/Arm.java [hunk 2]
```diff
@@ -179,9 +180,18 @@ public void periodic() {}
   /** Cancellation function */
   public void cancelScore() {
     goToPosition(ArmPosition.STARTING);
```
Comment (balghane) [inline src/main/java/frc/robot/subsystems/arm/Arm.java:182]: Maybe arm should stay in current position when cancelling, and it just shouldn't release the game piece

File: src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java [hunk 3]
```diff
@@ -519,6 +519,10 @@ public Command turnInPlace(double timeoutSec) {
         .withTimeout(timeoutSec);
   }
 
+  public void rezeroGyro() {
+    gyro.reset();
+  }
```
Comment (balghane) [inline src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java:524]: I think you can use `resetYaw`

File: src/main/java/frc/robot/RobotContainer.java [hunk 4]
```diff
@@ -34,6 +48,21 @@ public class RobotContainer {
   private TagLimelight tagLimelight = new TagLimelight();
   private DriveSubsystem drive = new DriveSubsystem(lights, () -> timedMatch);
 
+  private final CommandXboxController driverController =
+      new CommandXboxController(RobotMap.DRIVER_CONTROLLER_PORT);
+
+  Command rumbleBriefly =
```
[the remark this answers is not in the document]
Comment (avakimc) [reply src/main/java/frc/robot/RobotContainer.java:54]: i will make it used when the sensor sees a game piece 👍

File: src/main/java/frc/robot/subsystems/arm/Arm.java (hunk 1 above)
Comment (avakimc) [reply src/main/java/frc/robot/subsystems/arm/Arm.java:46]: removed!

File: src/main/java/frc/robot/subsystems/arm/Arm.java [hunk 5]
```diff
@@ -178,18 +177,15 @@ public void periodic() {}
 
   /** Cancellation function */
   public void cancelScore() {
-    goToPosition(ArmPosition.STARTING);
-    isScoring = false;
+    setCancelScore(true);
```
Comment (voidSevenSevenSix) [inline src/main/java/frc/robot/subsystems/arm/Arm.java:190]: Directly assign this variable instead of using the setter.

**Review by balghane — APPROVED** [review]

**Pull request merged**